backend/app/routers/listeners.py

```diff
@@ -374,10 +391,66 @@ async def edit_listener(
     raise HTTPException(status_code=404, detail=f"listener {listener_id} not found")
 
 
+@router.put("/{listener_id}/enable", response_model=EventListenerOut)
+async def enable_listener(
+    listener_id: str,
+    user_id=Depends(get_user),
+    allow: bool = Depends(ListenerAuthorization()),
+):
+    """Enable an Event Listener. Only admins can enable listeners.
+
+    Arguments:
+        listener_id -- UUID of the listener to be enabled
+    """
+    return await set_active_flag(listener_id, True)
+
+
+@router.put("/{listener_id}/disable", response_model=EventListenerOut)
+async def disable_listener(
+    listener_id: str,
+    user_id=Depends(get_user),
+    allow: bool = Depends(ListenerAuthorization()),
+):
+    """Disable an Event Listener. Only admins can enable listeners.
+
+    Arguments:
+        listener_id -- UUID of the listener to be enabled
```

Comment:
These two endpoints seems a little bit redundant? Could it be just one endpoint accepting the parameter then decide to activate or de-activate

Reply:
Please see my reply to Luigi above. 

If you all think this is redundant, I will get rid of these two.